lib/brakes/brakes.cpp

```diff
@@ -0,0 +1,16 @@
+#include "brakes.hpp"
+
+brakes::Brakes(const std::uint8_t pin, const io::IGpio gpio) : pin_(pin)

```

Comment:
`Brakes::Brakes` and same for all `brakes::` should be `Brakes::`

Reply:
done